notebooks/ch16/classification_cost.md

```diff
@@ -195,4 +270,4 @@ The surprisingly simple gradient expression allows us to fit a logistic model to
 
 ## Summary
 
-The cross entropy cost that we use to fit the logistic model. Since it is a convex function, we use gradient descent to fit the model to the cost. We now have the necessary components of logistic regression: the model, cost function, and minimization procedure.
+The cost function for logistic models is based on cross-entropy loss. Since it is a convex function, we use gradient descent to fit the model to the cost. We now have the necessary components of logistic regression: the model, cost function, and minimization procedure.
```

Comment:
nit: The cost function isn't based on cross-entropy loss, it *is* cross-entropy loss. How about:

> Since the cross-entropy loss function is convex, we minimize this loss using gradient descent to fit logistic models to data.